sdk/communication/cspell.json

```diff
@@ -18,7 +18,8 @@
         "deserializers",
         "ctxt",
         "wday",
-		"unhold",
-        "Unhold"
+        "unhold",
+        "Unhold",
+        "myinlineimage"
```

Comment:
in most places except one you're using `my-inline-image`, consider changing `myinlineimage` to that snake-case and then you don't need to add it here.